worker/src/ocall_bridge/bridge_api.rs

```diff
@@ -140,3 +166,14 @@ pub trait WorkerOnChainBridge {
         signed_blocks: Vec<u8>,
     ) -> OCallBridgeResult<()>;
 }
+
+/// type for IPFS
+pub type Cid = [u8; 46];
+
+/// Trait for all the OCalls related to IPFS
+#[cfg_attr(test, automock)]
+pub trait IpfsBridge {
+    fn write_to_ipfs(&self, data: &'static [u8]) -> OCallBridgeResult<Cid>;
```

Comment:
Is static needed here? Can't we just use `&[u8]`?

Reply:
I know that it has been like that before. I am just wondering. :)

Reply:
I tried - but unfortunately the `IpfsClient` (third-party library) defines its `add()` method as:
```
 #[inline]
    pub async fn add<R>(&self, data: R) -> Result<response::AddResponse, Error>
    where
        R: 'static + Read + Send + Sync,
    {
        self.add_with_options(data, request::Add::default()).await
    }
```

Reply:
Ah I see. Good to know!